RackAndPower/PowerShelfManagementSpec.md

```diff
@@ -0,0 +1,644 @@
+# Scope
+
+This document references requirements and provide the usage examples for the OCP Power Shelf Management API v1.0.0.
+
+# Requirements
+
+As a Redfish-based interface, the required Redfish interface model elements are specified in a profile document.
+For the Power Shelf Management API v1.0.0, the profile is located at: <TBD>
+
+The Redfish Interop Validator is an open-source conformance test that reads the profile, executes the tests against an implementation, and generates a test report in text or HTML format.
+
+```
+> python3 RedfishInteropValidator.py -u user -p password -r host:port profileName
+```
+
+The Redfish Interop Validator is located at https://github.com/DMTF/Redfish-Interop-Validator.
+
+The Power Shelf Management v1.0.0 profile extends from the Baseline Hardware Management v1.0.1 profile.
+This extension is specified directly in the profile.
+This means that the switch profile specifies conformance to the baseline profile in addition to any requirements specified in the Power Shelf Management profile.
+
+```
+"RequiredProfiles": {
+    "OCPBaselineHardwareManagement": {
+        "MinVersion": "1.0.1"
+    }
+},
+```
+
+# Capabilities
+
+The following use cases are enabled by conformance to this Power Shelf Management profile.
+The Power Shelf Management profile is extended from the Baseline Hardware Management profile.
+The following table lists the capabilities provide the baseline profile requirements.
+These capabilities are described in the "Usage Guide and Requirements for the OCP Baseline Hardware Management Profile v1.0.1" document.
+

```

Comment:
The last column should contain a link/ref to the section showing the usage.

Reply:
I made embedded links to the use case sections for the power shelf-specific use cases. How should we do this for the use cases covered by the baseline profile? I don't see a baseline doc to reference.